docs/configuration/collections.mdx

```diff
@@ -158,6 +158,7 @@ The following options are available:
 | **`beforeListTable`**      | An array of components to inject _before_ the built-in List View's table                                                                   |
 | **`afterList`**            | An array of components to inject _after_ the built-in List View                                                                            |
 | **`afterListTable`**       | An array of components to inject _after_ the built-in List View's table                                                                    |
+| **`listControlMenu`**      | An array of components to render as buttons within a menu next to the List Controls (after the Columns and Filters options)                |
```

Comment:
Since these are technically _menu items_, I wonder if `listMenuItems` is a better naming convention here.